pkg/apis/channels/v1alpha1/clusterbus_types.go

```diff
@@ -50,17 +50,20 @@ var _ webhook.GenericCRD = (*ClusterBus)(nil)
 type ClusterBusSpec = BusSpec
 
 // ClusterBusStatus (computed) for a clusterbus
-type ClusterBusStatus struct {
-}
+type ClusterBusStatus = BusStatus
```

Comment:
I see we have this pattern already with ClusterBusSpec but I think we'd want a new type rather than an alias here (e.g. type ClusterBusStatus BusStatus). I'm curious if there's a reason for the alias?

Reply:
This was also done to re-use some of the `bus_util` functions that deals with Status for ClusterBus. I agree `ClusterBusStatus` can be its own type when it diverges from `BusStatus`.